pkg/securitypolicy/securitypolicy_marshal.go

```diff
@@ -46,6 +46,8 @@ var policyRegoTemplate string
 //go:embed open_door.rego
 var openDoorRegoTemplate string
 
+var openDoorRego = strings.Replace(openDoorRegoTemplate, "@@API_SVN@@", apiSVN, 1)
+
```

Comment:
(unrelated) nit: it may be good to group all the embedded template strings and api version definitions in one place, and then in an init function replace all occurrences of the SVN placeholders.

Reply:
Adding to nit list.